test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -77,7 +79,7 @@ static inline void multi_consumer_pub_sub_test(bool intra_process)
   // test spin_some
   // Expectation: The message was published and all subscriptions fired the callback.
   // Use spin_once to block until published message triggers an event
-  //executor.spin_once();
+  // executor.spin_once();
```

Comment:
This is in the category of stuff I will clean up when I consider this test ready for review, which will be when the fixes for multithreading are ready for review.